lesson2/classwork.js

```diff
@@ -0,0 +1,28 @@
+function introduceMyselfWithName(name) {
+    console.log("name:", name);
+    console.log("age: 18");
+    console.log("education: pedic");
+    console.log("religion: serduckism");
+    console.log("credo: glory to Serduck and that's it");
+}
+
+function introduceMyself() {
+    introduceMyselfWithName("Gogi")
+}
+
+function introduceMyselfAssFemGogi() {
+    introduceMyselfWithName("FemGogi")
+}
+
+introduceMyselfWithName("Vasiyl")
+
+console.log("######################################");
+
+introduceMyself()
+
+introduceMyselfAssFemGogi()
+
+introduceMyselfWithName("Gogi")
+
+
+
```

Comment:
Leave only one of those blank lines. 
*Note: always end your text files with a blank line.*
*FYI: [why should text files end with a newline](https://stackoverflow.com/questions/729692/why-should-text-files-end-with-a-newline).*